### DISCUSSION TITLE
not working

### DISCUSSION THREAD

**Discussion by frkr** [dimaskiddo/go-whatsapp-multidevice-rest#42] [Q&A] [answered]
```
Versão 2.2330.11
```

```
docker run -d   -p 3000:3000 -e AUTH_BASIC_PASSWORD=1234 -e AUTH_JWT_EXPIRED_HOUR=0 
-e WHATSAPP_VERSION_MAJOR=2 -e WHATSAPP_VERSION_MINOR=2330 -e WHATSAPP_VERSION_PATCH=11 
 -e WHATSAPP_USER_AGENT_NAME=Chrome -e AUTH_JWT_SECRET=ThisIsJWTSecret -e HTTP_CORS_ORIGIN=* --name go-whatsapp-multidevice   dimaskiddo/go-whatsapp-multidevice-rest:v1.5.1  
```

```
Logging out WhatsApp Client for 551192012xxxx Due to Missmatch Authentication 
```

**Comment by ibnux** [comment, marked as answer]
To get JWT Auth

Phone number must be the same 

551192012xxxx:secret

then make and auth request to get QR scan it with whatsapp with the same number 551192012xxxx

https://github.com/dimaskiddo/go-whatsapp-multidevice-rest/wiki/Endpoint#auth

**Comment by frkr** [comment]
Yey... its working

![image](https://github.com/dimaskiddo/go-whatsapp-multidevice-rest/assets/185359/efefc946-24b8-4136-805f-47a532436e37)
